hknweb/academics/models/logistics.py

```diff
@@ -1,11 +1,16 @@
 from django.db import models
 from django.core.validators import MinValueValidator
 
+
 from .base_models import AcademicEntity
 
 
+
 class Course(AcademicEntity):
-    pass
+    CHILDREN = ["icsr"]
+    current_name = models.TextField(blank=True, null=True)

```

Comment:
Add these to the corresponding Serializer e.g. https://github.com/compserv/hknweb/blob/8707226e264c87e871a050c73134701dae244fa5/hknweb/academics/serializers/logistics_serializers.py#L9